plugins/basketball-scoreboard/basketball.py

```
@@ -9,12 +9,60 @@
+            url = f"https://site.api.espn.com/apis/site/v2/sports/{self.sport}/{self.league}/teams/{team_id}"
+            response = self.session.get(url, headers=self.headers, timeout=10)
+            response.raise_for_status()
+            team_data = response.json().get("team", {})
+            record_items = team_data.get("record", {}).get("items", [])
+            for item in record_items:
+                if item.get("type") == "total":
+                    record = item.get("summary", "")
+                    if record:
+                        self.cache_manager.set(cache_key, record)
+                        return record
+        except Exception as e:
```

Ruff (0.15.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)